src/features/common/types/project.d.ts

```diff
@@ -18,3 +18,247 @@ export interface SingleProject {
   currency: string;
   unitCost: number;
 }
+
+export interface SingleProjectGeojson {
+  geometry: Geometry
+  type: string
+  properties: Properties
+}
+
+export interface Geometry {
+  coordinates: number[][][]
+  type: string
+}
+
+export interface Properties {
+  lastUpdated: LastUpdated
+  name: string
+  description: string
+  id: string
+  status: string
+}
+
+export interface LastUpdated {
+  date: string
+  timezone: string
+  timezone_type: number
+}
+
+
+//single plant location 
+
+export interface SinglePlantLocation {
+  metadata: Metadata
+  hid: string
+  otherSpecies: any
```

Comment:
Could you replace any to specific data types?

Reply:
> @sunilsabatp Could you look into this? ![image](https://user-images.githubusercontent.com/72646230/194164190-51a43fd1-2ad6-4666-ad96-454aac1fd86c.png)

this issue have been fixed now